justfile

```diff
@@ -174,8 +174,8 @@ check-migrations: devenv
 
 # fix formatting and import sort ordering
 fix: devenv
-    $BIN/black .
     $BIN/ruff check --fix .
+    $BIN/ruff format .
```

Comment:
What does `check --fix` do that `format` doesn't do?

Reply:
`ruff check` runs the linter and fixes any automatically fixable issues it finds (example: removing unused imports), but `ruff check` doesn't run the formatter.

It's planned for `ruff` to [eventually unify the two](https://github.com/astral-sh/ruff/issues/8232).

Reply:
I need to read up on the difference between linting and formatting!